### ISSUE TITLE
Elixir style guide and identifier names

### ISSUE THREAD

**Issue by tmbb** [OvermindDL1/ex_spirit#3]
The function `repeatFn` should be renamed to `repeat_fn` according to the elixir style guide.

**Comment by OvermindDL1** [comment]
> The function repeatFn should be renamed to repeat_fn according to the elixir style guide.

Yeah I was doing too much coding in other languages at the time and it leaked over....  ^.^;

That will be a breaking change though so I'll wait for that on a major version bump, leaving this open for now to remind me (I'll probably have both in a minor bump then remove the `repeatFn` one on a major bump).

**Comment by tmbb** [comment]
There are some other identifiers no written according to the elixir style guide that aren't user-visible. I might submit a PR one of these days.

**Comment by OvermindDL1** [comment]
> There are some other identifiers no written according to the elixir style guide that aren't user-visible. I might submit a PR one of these days.

That'd be cool, just keep the old names for now too but mark them as deprecated for later removal.  :-)
At the very least if you want to document them all that would be great, or I can use a regex over it sometime like `[a-z]+[A-Z]` to find them.  ^.^
